nucypher/policy/conditions/lingo.py

```diff
@@ -104,21 +105,46 @@ def __init__(self, comparator: str, value: Any):
 
         self.comparator = comparator
         self.value = value
+        self.key = key
```

Comment:
This is a good area to prevent an invalid `ReturnValue` state by enforcing a valid type of `key`.

Reply:
Bumping this one - perhaps we can type-check `key` in case it's an invalid type here to promote a fast failure:

```python
if not any(isinstance(self.k, t) for t in (str, int)): raise
```

Reply:
👍 Nice - Good to catch this early